packages/features/ee/payments/api/webhook.ts

```diff
@@ -180,10 +265,56 @@ async function handlePaymentSuccess(event: Stripe.Event) {
   });
 }
 
+const handleSetupSuccess = async (event: Stripe.Event) => {
+  const setupIntent = event.data.object as Stripe.SetupIntent;
+  const payment = await prisma.payment.findFirst({
+    where: {
+      externalId: setupIntent.id,
+    },
+  });
+
+  if (!payment?.data || !payment?.id) throw new HttpCode({ statusCode: 204, message: "Payment not found" });
+
+  const { booking, user, evt, eventTypeRaw } = await getBooking(payment.bookingId);
+
+  const bookingData: Prisma.BookingUpdateInput = {
+    paid: true,
```

Comment:
This may be incorrect 💭

Reply:
On line 317, this function only triggers when we get a setup intent succeed event from Stripe